geomesa-accumulo/geomesa-accumulo-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -50,13 +54,16 @@ declare -a urls=(
   "${base_url}org/apache/hadoop/hadoop-hdfs/${hadoop_version}/hadoop-hdfs-${hadoop_version}.jar"
   "${base_url}commons-logging/commons-logging/${com_log_version}/commons-logging-${com_log_version}.jar"
   "${base_url}org/apache/commons/commons-vfs2/${commons_vfs2_version}/commons-vfs2-${commons_vfs2_version}.jar"
+  "${base_url}org/apache/htrace/htrace-core/${htrace3_core_version}/htrace-core-${htrace3_core_version}.jar"
 )
 
-# compare the first digit of htrace core version to determine the artifact name
-if [[ "${htrace_core_version%%.*}" -lt 4 ]]; then
-  urls+=("${base_url}org/apache/htrace/htrace-core/${htrace_core_version}/htrace-core-${htrace_core_version}.jar")
-else
-  urls+=("${base_url}org/apache/htrace/htrace-core4/${htrace_core_version}/htrace-core4-${htrace_core_version}.jar")
+
+# hadoop 2 minor version
+hmin=$(echo $hadoop_version | sed -e 's/[0-9]\+\.\([0-9]\+\)\.[0-9]\+/\1/')
+
+# if hadoop 2.8.x or higher also add htrace4 in addition to htrace3
+if [[ "${hmin}" -ge 8 ]]; then
```

Comment:
What's gonna happen with Hadoop 3.x clusters?!?!??!

Reply:
nothing yet. make a ticket. geomesa 3.0

Reply:
ok fine i'll match it.